src/Bundle/DependencyInjection/Compiler/CsrfTokenManagerPass.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ResourceBundle\DependencyInjection\Compiler;
+
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+
+/**
+ * TODO Remove on sylius/resource-bundle 2.0
+ */
+final class CsrfTokenManagerPass implements CompilerPassInterface
```

Comment:
It would be just the art for the sake of art, but we could try to cover this pass with configuration test as well :P